lib/utilities/mk-tmp-dir-in.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var Promise   = require('../ext/promise');
+var fs        = require('fs-extra');
+var temp      = require('temp');
+var mkdir     = Promise.denodeify(fs.mkdir);
+var mkdirTemp = Promise.denodeify(temp.mkdir);
+
+function exists(dir) {
+  return new Promise(function(resolve) {
+    fs.exists(dir, resolve);
```

Comment:
`fs.exists` has a different signature - the callback receives only one argument - the boolean value.